api/api_keys/models.py

```diff
@@ -18,3 +20,13 @@ class MasterAPIKey(AbstractAPIKey, SoftDeleteObject):
 
     objects = MasterAPIKeyManager()
     is_admin = models.BooleanField(default=True)
+
+    if settings.IS_RBAC_INSTALLED:
```

Comment:
This should be inside the method itself.

Reply:
Moved.